docs/operate/get-started/setup.md

```diff
@@ -30,35 +24,31 @@ aliases:
   - /get-started/installation/
 ---
 
-Get started by installing the open-source software that drives your hardware and connects your device to the cloud.
-The easiest way to do this is through the Viam app, so that your machines are automatically connected to configuration and remote operation tools.
+Get started by installing [`viam-server`](/operate/reference/viam-server/), the open-source software that drives your hardware and connects your device to the cloud.
 
-{{< expand "Supported systems" >}}
+## Prerequisite: Make sure you have a supported operating system
 
-Viam can run on any computer that runs one of the following operating systems:
+`viam-server` can run on any computer that runs one of the following operating systems:
 
 - Linux 64-bit operating systems running on AArch64 (ARM64) or x86-64 architectures
 - macOS
 - Windows
-  - native: use native if you are using a WSL version prior to WSL 2 or need native USB support
-  - WSL: use WSL if you are using python modules or other linux dependencies
 
-Viam also offers a lightweight binary to support the following 32-bit microcontrollers:
+For 32-bit microcontrollers, see [Set up an ESP32](/operate/get-started/setup-micro/).
```

Comment:
this should be at the end of the section, not above the command you want them to run

Reply:
Yeah with the above coming out I will move it. If that was staying it, i worried people would scan that quickly and think no esp32 support. Removing!